src/controller/CHIPDeviceController.cpp

```diff
@@ -2259,6 +2259,7 @@ void DeviceCommissioner::ContinueReadingCommissioningInfo(const CommissioningPar
 {
     VerifyOrDie(mCommissioningStage == CommissioningStage::kReadCommissioningInfo);
 
+    // mReadCommissioningInfoProgress starts at 0 and counts the number of attributes we have read.
```

Comment:
This is specifically the number of attribute paths we have issued reads for, not the number of attribute values we got back, due to wildcarding...